gdx/src/com/badlogic/gdx/graphics/g2d/PixmapPacker.java

```diff
@@ -326,66 +322,61 @@ public synchronized void dispose () {
 	 * @return the TextureAtlas */
 	public synchronized TextureAtlas generateTextureAtlas (TextureFilter minFilter, TextureFilter magFilter, boolean useMipMaps) {

```

Comment:
Sure, but the function doesn't do anything. You get the same functionality with:
```
atlas = new TextureAtlas();
packer.updateTextureAtlas(atlas, min, mag, mipMaps);
```